integration-test/546-road-sort-keys-tunnel.py

```diff
@@ -50,7 +50,7 @@ def test_secondary_level_0(self):
         self.assert_has_feature(
             16, 16812, 24391, 'roads',
```

Comment:
If you updated this to 17, 33624, 48782 then the name would still be there, but it's fine this way.